README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+
+```bash
+mkdir myproject
+cd myproject
+```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since you haven't added any dependency yet.
+
+```bash
+corral init
+```
+
+3. Add a dependency. This is the way to tell Corral that your project depends on this and you want to include it when building your project.
+
+```bash
+corral add github.com/jemc/pony-inspect.git
+```
+
+4. Use that dependency in your code. Create a file `main.pony` with following code.
+
+```pony
+use "inspect"
```

Comment:
Given this name is set by `pony-inspect` and not simply derived from dropping `pony-` from the dependency's name, do we need to (or really feel it is the place here) to clarify how dependencies get their names?

This might very well even be a conversation for having `pony add` put the import name in `corral.json` so there is a clear place where one can look up within `corral` what names outside of stdlib are currently available without needing to look at how each dependency names itself.

Reply:
I think using an example other than pony-inspect would be good.

I would suggest we use

https://github.com/ponylang/valbytes
or
https://github.com/ponylang/peg

Reply:
Just had another newbie moment 😄 On https://github.com/ponylang/valbytes I was looking for some example code and also what will be the argument for coral add. Copy pasted it i.e.  `corral add github.com/ponylang/valbytes` and then did `corral fetch` but nothing happened. 

After few minutes, I realised that I am missing `.git` suffix 😄 I guess I need to make another pull request for that